src/test/java/roomescape/application/ReservationServiceTest.java

```diff
@@ -0,0 +1,143 @@
+        assertThat(response.time()).isEqualTo(
+                new ReservationTimeResponse(reservationTime.getId(), reservationTime.getStartAt().toString()));
+    }
+
+    @Test
+    void 같은일시_같은테마_예약이_존재하면_예약을_생성할_수_없다() {
+        ReservationTime reservationTime = reservationTimeDbFixture.예약시간_10시();
+        Theme theme = themeDbFixture.공포();
+
+        reservationService.createReservation(new ReservationCreateRequest(
+                ReserverNameFixture.한스.getName(),
+                ReservationDateFixture.예약날짜_25_4_22.getDate(),
+                reservationTime.getId(),
+                theme.getId()
+        ));
+
+        ReservationCreateRequest request = new ReservationCreateRequest(
+                ReserverNameFixture.한스.getName(),
+                ReservationDateFixture.예약날짜_25_4_22.getDate(),
+                reservationTime.getId(),
+                theme.getId()
+        );
+
+        assertThatThrownBy(() -> reservationService.createReservation(request))
+                .isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @Test
+    void 같은일시_다른테마_예약은_생성할_수_있다() {
+        ReservationTime reservationTime = reservationTimeDbFixture.예약시간_10시();
+        Theme horror = themeDbFixture.공포();
+        Theme mystery = themeDbFixture.커스텀_테마("미스테리");
+
+        reservationService.createReservation(new ReservationCreateRequest(
+                ReserverNameFixture.한스.getName(),
+                ReservationDateFixture.예약날짜_25_4_22.getDate(),
+                reservationTime.getId(),
+                horror.getId()
+        ));
+
+        ReservationCreateRequest request = new ReservationCreateRequest(
+                ReserverNameFixture.한스.getName(),
+                ReservationDateFixture.예약날짜_25_4_22.getDate(),
+                reservationTime.getId(),
+                mystery.getId()
+        );
+
+        assertThatCode(() -> reservationService.createReservation(request))
+                .doesNotThrowAnyException();
+    }
+
+    @Test
+    void 예약을_모두_조회한다() {
+        ReservationTime reservationTime = reservationTimeDbFixture.예약시간_10시();
+        Theme theme = themeDbFixture.공포();
+
+        reservationService.createReservation(new ReservationCreateRequest(
+                ReserverNameFixture.한스.getName(),
+                ReservationDateFixture.예약날짜_25_4_22.getDate(),
+                reservationTime.getId(),
+                theme.getId()
+        ));
+        List<ReservationResponse> responses = reservationService.getReservations();
+        ReservationResponse response = responses.get(0);
+
+        assertThat(response.id()).isEqualTo(1L);
+        assertThat(response.name()).isEqualTo(ReserverNameFixture.한스.getName());
+        assertThat(response.date()).isEqualTo(ReservationDateFixture.예약날짜_25_4_22.getDate());
+        assertThat(response.time()).isEqualTo(
+                new ReservationTimeResponse(reservationTime.getId(), reservationTime.getStartAt().toString()));
```

Comment:
다건 조회 케이스는 조회 결과의 size도 검증해주는 것이 좋다고 생각해요. 😃
```suggestion

        assertThat(response.size).isEqualTo(1);
        assertThat(response.get(0).id()).isEqualTo(1L);
        assertThat(response.get(0).name()).isEqualTo(ReserverNameFixture.한스.getName());
        assertThat(response.get(0).date()).isEqualTo(ReservationDateFixture.예약날짜_25_4_22.getDate());
        assertThat(response.get(0).time()).isEqualTo(
                new ReservationTimeResponse(reservationTime.getId(), reservationTime.getStartAt().toString()));
```

Reply:
꼼꼼하게 리뷰해주셔서 감사해요 로키!
말씀해주신 내용대로 다건 조회 케이스는 조회 결과의 size까지 검증해주는 것이 명확한 테스트가 되겠네요!
`ReservationTimeService`, `ThemeService`의 조회 테스트에서도 추가해주었습니다.😄

[refactor: 다건 조회 케이스 size 검증 추가](https://github.com/ljhee92/spring-roomescape-member/commit/0de9ef6a0d00864bc2ce5a2119fe70d774cd0f17)
[refactor: 여러 개의 테스트 assertAll로 수정](https://github.com/ljhee92/spring-roomescape-member/commit/b74736463cc9bf6728dc944b3bc3cebfd7ae1578)